docs/rest.md

```diff
@@ -330,6 +341,66 @@ mutation encryptedPost(
 
 [Unit Test](https://github.com/apollographql/apollo-link-rest/blob/c9d81ae308e5f61b5ae992061de7abc6cb2f78e0/src/__tests__/restLink.ts#L1847-L1904)
 
+<h5 id="rest.arguments.body.serializer">`bodySerializer`</h5>
+
+If you need to serialize your data differently (say as form-encoded), you can provide a `bodySerializer` instead of relying on the default JSON serialization.
+`bodySerializer` can be either a function of the form `(data: any, headers: Headers) => {body: any, header: Headers}` or a string key. When using the string key
+`RestLink` will instead use the corresponding serializer from the `bodySerializers` object that can optionally be passed in during initialization.
+
+```graphql
+mutation encryptedForm(
+  $input: PublishablePostInput!,
+  formSerializer: any
```

Comment:
This is just a question for my own sake: Why is there no `$` in front of this variable. Does that tell GraphQL not to serialize it (since it's a function)? Would it still be passed in on the `variables` same as the input value?

Reply:
I'm also curious! :)

Reply:
Ah lol, that would be a typo.

Reply:
O, right... I just thought it mightn't be because it was like that twice. Copy & past I guess. :smile: